ocs_ci/helpers/cephfs_stress_helpers.py

```
@@ -103,13 +107,20 @@ def setup_stress_test_environment(self, pvc_size, storageclass_factory):
-        )
+        try:
+            sc_obj = storageclass_factory(
+                sc_name=sc_name,
+                interface=constants.CEPHFILESYSTEM,
+                kernelMountOptions='context="system_u:object_r:container_file_t:s0"',
+            )
+            logger.info(f"Created SELinux CephFS StorageClass: {sc_name}")
+        except CommandFailed as e:
+            if "(AlreadyExists)" not in str(e):
+                raise
+            logger.warning(f"StorageClass '{sc_name}' already exists — reusing it")
```

Ruff (0.16.3)

[warning] Logging statement uses f-string

(G004)